man/ukify.xml

```diff
@@ -0,0 +1,309 @@
+
+  <refentryinfo>
+    <title>ukify</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>ukify</refentrytitle>
+    <manvolnum>1</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>ukify</refname>
+    <refpurpose>Combine kernel and initrd into a signed Unified Kernel Image</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <cmdsynopsis>
+      <command>/usr/lib/systemd/ukify</command>
+      <arg choice="plain"><replaceable>LINUX</replaceable></arg>
+      <arg choice="plain" rep="repeat"><replaceable>INITRD</replaceable></arg>
+      <arg choice="opt" rep="repeat">OPTIONS</arg>
+    </cmdsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para>Note: this command is experimental for now. While it is intended to become a regular component of
+    systemd, it might still change in behaviour and interface.</para>
+
+    <para><command>ukify</command> is a tool that combines a kernel and an initrd with
+    a UEFI boot stub to create a
+    <ulink url="https://uapi-group.org/specifications/specs/unified_kernel_image/">Unified Kernel Image (UKI)</ulink>
+    — a PE binary that can be executed in UEFI context to start the embedded linux kernel.
+    See <citerefentry><refentrytitle>systemd-stub</refentrytitle><manvolnum>7</manvolnum></citerefentry>
+    for details about the stub.</para>
+
+    <para>Additional sections will be inserted into the UKI, either automatically or only if a specific
+    option is provided. See the discussion of
+    <option>--cmdline=</option>,
+    <option>--os-release=</option>,
+    <option>--devicetree=</option>,
+    <option>--splash=</option>,
+    <option>--pcrpkey=</option>,
+    <option>--uname=</option>,
+    and <option>--section=</option>
+    below.</para>
+
+    <para>If PCR signing keys are provided via the <option>--pcr-public-key=</option> and
+    <option>--pcr-private-key=</option> options, PCR values that will be seen after booting with the given
+    kernel, initrd, and other sections, will be calculated and signed and embedded in the UKI.
+    <citerefentry><refentrytitle>systemd-measure</refentrytitle><manvolnum>1</manvolnum></citerefentry> is
+    used to perform this calculation and signing.</para>
+
+    <para>The calculation of PCR values is done for specific boot phase paths. Those can be specified with
+    <option>--phases=</option> option. If not specified, the default provided by
+    <command>systemd-measure</command> is used. If is also possible to specify the
+    <option>--pcr-private-key=</option>, <option>--pcr-public-key=</option>, and <option>--phases=</option>
+    arguments more than once. Signatures will be then performed with each of the specified keys. When both
+    <option>--phases=</option> and <option>--pcr-private-key=</option> are used, they must be specified the
+    same number of times, and then n-th boot phase path set will be signed by the n-th key. This can be used
```

Comment:
I think "n-th" is easier to pronounce. "k-th key" is a bit of a mouthful.

"then" seems OK to me. You specify the keys, and then (later) something is done with them.

Reply:
I think "then" there is ok, too, but then it's still missing an article, I think:
```suggestion
    same number of times, and then the n-th boot phase path set will be signed by the n-th key. This can be used
```